libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/EtterlatteBrevkode.kt

```diff
@@ -79,7 +79,17 @@ enum class Brevtype {
     NOTAT,
     OVERSENDELSE_KLAGE,
     SLETTET_VARSEL,
+    INGEN, // Til bruk for å modellere at brevet ikke har en redigerbar del
     ;
 
     fun erKobletTilEnBehandling(): Boolean = this in listOf(VEDTAK, VARSEL, VEDLEGG)
 }
+
+fun tittelForVedlegg(kode: EtterlatteBrevKode) =
```

Comment:
Denn funksjonen er vell kun tenkt for enumer som har `erVedlegg = true` ?, 

Bør vi iikke derfor sjekke på det? 

Eksempel
```
fun tittelForVedlegg(kode: EtterlatteBrevKode): String {
    return if (kode.erVedlegg) {
          when (kode) {
            ...
          }
        } else {
        throw IllegalArgumentException("Denne funksjonen er bare for vedlegg, $kode er ikke et vedlegg")
    }
```